### PR TITLE
Adds scaffolding configs to improve cross-platform dev consistency

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
```
Comment (YakDriver) [inline .gitattributes]: 👍

**Comment by lorengordon** [comment]
@YakDriver good point... https://github.com/plus3it/terraform-aws-watchmaker/pull/98/commits/75e661ea1795d7bf9b48e85a51492251052031fb

**Comment by lorengordon** [comment]
@YakDriver ok, this is good for another look

**Comment by lorengordon** [comment]
go codebuild go

**Pull request merged**